services/blockchain-indexer/shared/indexer/transactionProcessor/interoperability/submitSidechainCrossChainUpdate.js

```diff
@@ -33,13 +36,27 @@ const getBlockchainAppsTable = () => getTableInstance(
 	MYSQL_ENDPOINT,
 );
 
+const getCCUTable = () => getTableInstance(
+	ccuTableSchema.tableName,
+	ccuTableSchema,
+	MYSQL_ENDPOINT,
+);
+
+const getTransactionsTable = () => getTableInstance(
+	transactionsTableSchema.tableName,
+	transactionsTableSchema,
+	MYSQL_ENDPOINT,
+);
```

Comment:
With the latest mysql updates, we don't need to specify the table name anymore.
```suggestion
const getCCUTable = () => getTableInstance(ccuTableSchema, MYSQL_ENDPOINT);
const getTransactionsTable = () => getTableInstance(transactionsTableSchema, MYSQL_ENDPOINT);
```